### PR TITLE
Update PromQL readme

### PR DESCRIPTION
**Pull request by RichiH**

### CODE REVIEW

File: promql/README.md [hunk 1]
```diff
@@ -50,6 +50,6 @@ It's still early days for the PromQL Compliance Tester. In particular, we would
 * Test more vendor implementations of PromQL.
 * Version test results and make pretty output presentations easier.
 
-**Note:** Many people will be interested in benchmarking performance differences between PromQL implementations. While this is important as well, the PromQL Compliance Tester focuses solely on correctness testing.
+**Note:** Many people will be interested in benchmarking performance differences between PromQL implementations. While this is important as well, the PromQL Compliance Tester focuses solely on correctness testing. Please contact the [maintainers](../MAINTAINERS.md) if you want work on performance testing.
```
Comment (juliusv) [inline promql/README.md:53]:
```suggestion
**Note:** Many people will be interested in benchmarking performance differences between PromQL implementations. While this is important as well, the PromQL Compliance Tester focuses solely on correctness testing. Please contact the [maintainers](../MAINTAINERS.md) if you want to work on performance testing.
```

...but not sure we should even include that in this repo at all. I see this repository being 100% about compliance and no other performance characteristics (unless of course they make something "funktional kaputt").

Comment (RichiH) [reply promql/README.md:53]: I do think that a baseline of performance will be needed at some point. The performance in our ecosystem has set expectations and while we can't, and shouldn't, demand super high performance, an engine which can only process one datapoint per second would not be compliant in fact.

Comment (juliusv) [reply promql/README.md:53]: @RichiH Yeah, I would cover that under "funktional kaputt". But ok, let's keep this in for now.

**Comment by juliusv** [comment]
:+1:

**Pull request merged**